packages/react-components/src/types/OnRender.ts

```diff
@@ -36,7 +36,7 @@ export type CustomAvatarOptions = {
   text?: string;
   /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
   /** State for the participant to be displayed in the defaultPlaceHolder */
-  state?: ParticipantState;
+  participantState?: ParticipantState;
```

Comment:
Why the name change in this PR? Let's keep this PR focused on tests. Also, this is, unfortunately, called `state` in headless SDK, so perhaps we want to keep the naming aligned even in the components API?

Reply:
Recommend against making it `state`.
It is too generic. Moreover, we have used the term `participantState` in other components as well.

Reply:
I also like `participantState` better, especially if we have precedent.